pkg/pillar/containerd/oci.go

```diff
@@ -217,3 +221,29 @@ func (s *ociSpec) updateFromImageConfig(config v1.ImageConfig) error {
 	}
 	return oci.WithAdditionalGIDs("root")(ctrdCtx, CtrdClient, &dummy, &s.Spec)
 }
+
+// UpdateMounts
+func (s *ociSpec) UpdateMounts(disks []types.DiskConfig) {
+	for i, disk := range disks {
```

Comment:
would't it be better to simply iterate over range disks[1:] ?

Reply:
Resolved